test/models/door_test.exs

```diff
@@ -33,7 +33,7 @@ defmodule Gatekeeper.DoorTest do
     company = create_company
     member = create_member company: company, door_group: door_group
 
-    assert Door.member_access_allowed? door, member
+    assert {true, _reason} = Door.member_access_allowed? door, member
```

Comment:
I don't really care about enforcing whether or not the variable is read. The underscore is there primarily to silence compiler warnings, and I left it with a name partially for documentation purposes. If I want to make an assertion on that value in the future, I already know the intent of the value.